aim/web/ui/src/pages/MetricsExplorer/config.tsx

```diff
@@ -81,37 +78,24 @@ export const getMetricsDefaultConfig = (): typeof defaultHydration => {
       component: Aggregation,
       state: {
         initialState: {
-          methods: {
-            area: AggregationAreaMethods.MIN_MAX,
-            line: AggregationLineMethods.MEAN,
-          },
-          isApplied: false,
+          methods: CONTROLS_DEFAULT_CONFIG.metrics.aggregationConfig.methods,
+          isApplied:
+            CONTROLS_DEFAULT_CONFIG.metrics.aggregationConfig.isApplied,
+        },
+        persist: PersistenceTypesEnum.Url,
+      },
+    };
+    draft.axesProperties = {
+      component: ConfigureAxes,
+      state: {
+        initialState: {
+          alignment: CONTROLS_DEFAULT_CONFIG.metrics.alignmentConfig,
+          axesScaleType: CONTROLS_DEFAULT_CONFIG.metrics.axesScaleType,
+          axesScaleRange: CONTROLS_DEFAULT_CONFIG.metrics.axesScaleRange,
         },
         persist: PersistenceTypesEnum.Url,
       },
     };
-    // draft.axesProperties = {
-    //   component: () => null,
-    //   state: {
-    //     initialState: {
-    //       alignment: {
-    //         metric: CONTROLS_DEFAULT_CONFIG.metrics.alignmentConfig.metric,
-    //         type: CONTROLS_DEFAULT_CONFIG.metrics.alignmentConfig.type,
-    //       },
-    //       axesScale: {
-    //         type: {
-    //           xAxis: CONTROLS_DEFAULT_CONFIG.metrics.axesScaleType.xAxis,
-    //           yAxis: CONTROLS_DEFAULT_CONFIG.metrics.axesScaleType.yAxis,
-    //         },
-    //         range: {
-    //           xAxis: CONTROLS_DEFAULT_CONFIG.metrics.axesScaleRange.xAxis,
-    //           yAxis: CONTROLS_DEFAULT_CONFIG.metrics.axesScaleRange.yAxis,
-    //         },
-    //       },
-    //     },
-    //     persist: 'url',
-    //   },
-    // };
```

Comment:
What about adding TODO here or deleting it?

Reply:
it's already removed thanks